applications/FluidDynamicsApplication/symbolic_generation/two_fluid_navier_stokes/generate_two_fluid_navier_stokes.py

```diff
@@ -111,32 +112,74 @@
         vconv_gauss_norm += vconv_gauss[i]**2
     vconv_gauss_norm = sqrt(vconv_gauss_norm)
 
-    tau1 = 1.0/((rho*dyn_tau)/dt + (stab_c2*rho*vconv_gauss_norm)/h + (stab_c1*mu)/(h*h) + K_darcy)   # Stabilization parameter 1
-    tau2 = mu + (stab_c2*rho*vconv_gauss_norm*h)/stab_c1                                    # Stabilization parameter 2
+    ## Data interpolation to the Gauss points
+    if time_integration=="bdf2":
+        K_darcy = Symbol('K_darcy', positive = True)
```

Comment:
Why are you not including the darcy term in the new formulation?

Reply:
Because it is not required for @uxuech applications. We can include it in the future if required.

Reply:
If it requires some special treatment we can leave it for the future, if its more or less straight away, I'd add it now.

Reply:
We're a bit on a hurry as we need this to be merged to continue ongoing developments. We could eventually add it in a future PR.